contracts/sg-ics721/src/testing/integration_tests.rs

```diff
@@ -927,6 +927,7 @@ fn test_do_instantiate_and_mint() {
         assert_eq!(base_owner, owner);
     }
     // test case: instantiate cw721 with CustomClassData (without owner, name, and symbol)
+    // as a result instantiated collection on target chain will use class id for name and symbol
```

Comment:
This is the test, in case custom data is passed to target chain (e.g. SgCollectionData). So target chain wont have name and symbol.